dora-metrics/pelorus
# Rebuilding docs requirements tree to get the builds working again.

## Description

This PR pulls in updated versions of all the packages involved in building our docs with `mkdocs`. This should fix the broken builds on readthedocs.org

## Testing Instructions

1. Check the readthedocs CI check and make sure that it passes
2. Check the built docs site and make sure everything looks right
